curriculum/challenges/english/25-front-end-development/lecture-best-practices-for-styling-forms/672bc9d9eecb65cdf63491de.md

```diff
@@ -10,6 +10,18 @@ dashedName: when-should-you-use-appearance-none-to-deal-with-issues-styling-sear
 
 Watch the lecture video and answer the questions below.
 
+# --transcript--
+
+When should you use `appearance: none` to deal with issues styling search inputs and checkboxes?
+
+Let's learn when to use `appearance: none` to address styling issues for search input, checkboxes and radio buttons.
+
+Browsers apply default styling to a lot of elemets, in the case of input elements your ability to style them with CSS can feel rather restricted. So, you may want to use `appearance: none` to hide aspects of the default element and build your own instead. For example, this would hide the default checkboxes for a checkbox input allowing you to use custom indicators like a green tick and a red X to show the state. For a search input, webkit based browsers will show a default search icon and a cancel button. Hiding these allows you to create your own indicators that would appear on all browsers. 
```

Comment:
```suggestion
Browsers apply default styling to a lot of elements. In the case of input elements your ability to style them with CSS can feel rather restricted. So, you may want to use `appearance: none` to hide aspects of the default element and build your own instead. For example, this would hide the default checkboxes for a checkbox input allowing you to use custom indicators like a green tick and a red X to show the state. For a search input, WebKit based browsers will show a default search icon and a cancel button. Hiding these allows you to create your own indicators that would appear on all browsers. 
```